### PR TITLE
Removes farting from Self respiration symptom

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

title

## Why It's Good For The Game

cringe fetish

## Changelog
:cl:
del: Removed farting from Self respiration
/:cl:

### CODE REVIEW

**Comment by amidoingitright** [comment]
based and katpilled

**Comment by TheFakeElon** [comment]
you can add /* to start a comment and */ to end one, that way you don't have to // every line, makes things more neat

**Comment by zeskorion** [comment]
ok felid player

**Comment by totallynotkatfood** [comment]
ok blasman player

File: code/datums/diseases/advance/symptoms/oxygen.dm [hunk 1]
```diff
@@ -29,9 +29,9 @@ Bonus
 	symptom_delay_min = 1
 	symptom_delay_max = 1
 	var/regenerate_blood = FALSE
-	var/gas_type = /datum/gas/miasma
-	var/base_moles = 3
-	var/emote = "fart"
+	//var/gas_type = /datum/gas/miasma austation begin -- fetish content
+	//var/base_moles = 3
+	//var/emote = "fart" austation end
```
Comment (TheFakeElon) [inline code/datums/diseases/advance/symptoms/oxygen.dm:34]:
```suggestion
/* austation begin -- fetish content
	var/gas_type = /datum/gas/miasma
	var/base_moles = 3
	var/emote = "fart"
austation end */
```

File: code/datums/diseases/advance/symptoms/oxygen.dm [hunk 2]
```diff
@@ -51,14 +51,14 @@ Bonus
...
-				T.air_update_turf()
-				M.emote(emote)
+			//if(prob(1) && prob(70)) austation begin -- fetish content
+			//	var/turf/open/T = get_turf(M)
+			//	if(!istype(T))
+			//		return
+			//	var/datum/gas_mixture/air = T.return_air()
+			//	air.set_moles(gas_type, air.get_moles(gas_type) + base_moles)
+			//	T.air_update_turf()
+			//	M.emote(emote) austation end
```
Comment (TheFakeElon) [inline code/datums/diseases/advance/symptoms/oxygen.dm:61]:
```suggestion
/* austation begin -- fetish content
			if(prob(1) && prob(70))
				var/turf/open/T = get_turf(M)
				if(!istype(T))
					return
				var/datum/gas_mixture/air = T.return_air()
				air.set_moles(gas_type, air.get_moles(gas_type) + base_moles)
				T.air_update_turf()
				M.emote(emote)
austation end */

```

**Comment by MCterra10** [comment]
Imagine camping on your downstream's github repo instead of maintaining your own @zeskorion
It's almost like we're better or something 👉😎👉

**Comment by coolsurf6** [comment]
It seems Kat has never played Goon before, cringe.

**Pull request merged**